flow/designs/sky130hd/aes/config.mk

```diff
@@ -6,11 +6,14 @@ export VERILOG_FILES = $(sort $(wildcard ./designs/src/$(DESIGN_NICKNAME)/*.v))
 export SDC_FILE      = ./designs/$(PLATFORM)/$(DESIGN_NICKNAME)/constraint.sdc
 export RCX_RULES     = $(PLATFORM_DIR)/rcx_patterns.rules
```

Comment:
I just removed these `RCX_RULES` from the designs `config.mk` on #86, please make sure that this PR will not re-introduce them -- same for all other occurrences.

Reply:
Is you PR already merged into master?

Reply:
Yes

Reply:
@ravi-varadarajan you will have to update your sky* results as having RCX changes the final results.

Reply:
Should be ready for merge --removed the RCX settings from the design config file as @vvbandeira had already added it to the platform config file (and removed from design config file at the same time)